### PR TITLE
[BACKLOG-32444] Update "Job" job entry to use enhanced VFS browser fo…

### PR DESCRIPTION
**Pull request by ddiroma**
…r browsing and selecting local files

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 10m 46s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl ui
```
---
:ok_hand: All tests passed!
 
Tests run: **1123**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-7094/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 10m 53s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl ui
```
---
:ok_hand: All tests passed!
 
Tests run: **1123**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-7094/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 11m 11s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl ui
```
---
:ok_hand: All tests passed!
 
Tests run: **1123**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-7094/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by ddiroma** [comment]
@pentaho/r2d2

File: ui/src/main/java/org/pentaho/di/ui/job/entries/job/JobEntryJobDialog.java [hunk 1]
```diff
@@ -218,15 +222,10 @@ public void widgetSelected( SelectionEvent arg0 ) {
       }
     } );
 
-    wbBrowse.addSelectionListener( new SelectionAdapter() {
-      public void widgetSelected( SelectionEvent e ) {
-        if ( rep != null ) {
-          selectJob();
-        } else {
-          pickFileVFS();
```
Comment (cseverino789) [inline ui/src/main/java/org/pentaho/di/ui/job/entries/job/JobEntryJobDialog.java]: Looks like pickFileVFS is now no longer called, which has additional logic . Also it appears to have an extra dialog which I'm not sure is required anymore. Will need to determine what is needed in that method

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 15m 40s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl ui
```
---
:ok_hand: All tests passed!
 
Tests run: **1123**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-7094/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 14m 48s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl ui
```
---
:ok_hand: All tests passed!
 
Tests run: **1123**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-7094/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request closed without merging**